pkgs/tools/text/html-tidy/default.nix

```diff
@@ -11,9 +11,18 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-ZMz0NySxzX2XHiqB8f5asvwjIG6kdIcq8Gb3EbAxBaU=";
   };
 
-  nativeBuildInputs = [ cmake libxslt/*manpage*/ ];
+  # https://github.com/htacg/tidy-html5/pull/1036
+  patches = (fetchpatch {
```

Comment:
Yes, agree, should be fixed on next update.
Just in case, we have some other occurances:
```
$ find pkgs -name "*.nix" | xargs cat | grep "patches = (fetch\|patches = fetch" | wc -l
12
```